src/Pages/SignIn/SignIn.js

```diff
@@ -29,7 +29,7 @@ export default function SignIn() {
     setPassword(data.get("password"));
   };
 
-  const getUserToken = () => {
+  const getUserToken = async () => {
```

Comment:
if using async await we should use try and catch block but if we using .then() then we dont need async await but what I have learnt is that async/await is preferred way however for now we dont need to worry about it. Just revert the code here and merge this branch. 

Just add the issue number its closing.

Reply:
I removed async as well